src/Cake.Tfs/PullRequest/TfsPullRequestSettings.cs

```diff
@@ -66,6 +66,44 @@ public TfsPullRequestSettings(TfsPullRequestSettings settings)
             this.ThrowExceptionIfPullRequestCouldNotBeFound = settings.ThrowExceptionIfPullRequestCouldNotBeFound;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TfsPullRequestSettings"/> class using environment variables.
```

Comment:
We should mention in the summary that the method is only indented to be used while running in an Azure Pipelines build.